tables/README.md

```diff
@@ -68,4 +72,5 @@ Table S5: Key Resources
 
 ## Zenodo Upload
 
-> This section is forthcoming.
+Files in this directory are named as: `data-figure-<figure number><figure panel>.csv` (or `.csv.gz` if compressed.)
```

Comment:
Naming suggestion, because of how my brain works... I don't know if this is really worth doing, but here we are:
`figure3a-data.csv` For supplementals, I would allow for capital letters, because the lower case looks very strange: `figure-S2b-data.csv`.

Leaving this as a standalone comment, but I'll review more as I go. But if you want to change these, I figure earlier better than later.

Reply:
I actually meant to go back and add that... I had it for the supplemental because that one really does need it, so everywhere makes sense...